x/auth/simulation/genesis_test.go

```diff
@@ -2,6 +2,7 @@ package simulation_test
 
 import (
 	"encoding/json"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
```

Comment:
Please format with `goimports` - or just run `make format`. Bonus points if you install the githook available in contrib so that formatting would be done automatically at pre-commit

Reply:
Done